presto-kudu/src/main/java/com/facebook/presto/kudu/KuduClientConfig.java

```diff
@@ -114,15 +115,27 @@ public KuduClientConfig setDisableStatistics(boolean disableStatistics)
         return this;
     }
 
-    public String getTenant()
+    public String getInferSchemaPrefix()
     {
-        return tenant;
+        return isInferSchemaEnabled() ? inferSchemaPrefix : null;
```

Comment:
What about `checkState(isInferSchemaEnabled())` instead? Or not having this logic here at all?

Reply:
This is only cosmetics for the listing of configuration values during startup of Presto.
Using something like checkState would result in strange logging output during startup:
```
2018-05-01T13:50:55.711+0545 INFORMATION PROPERTY                                  DEFAULT      RUNTIME             DESCRIPTION
2018-05-01T13:50:55.712+0545 INFORMATION kudu.client.defaultAdminOperationTimeout  30.00s       30.00s
2018-05-01T13:50:55.712+0545 INFORMATION kudu.client.defaultOperationTimeout       30.00s       30.00s
2018-05-01T13:50:55.712+0545 INFORMATION kudu.client.defaultSocketReadTimeout      10.00s       10.00s
2018-05-01T13:50:55.712+0545 INFORMATION kudu.client.disableStatistics             false        false
2018-05-01T13:50:55.713+0545 INFORMATION kudu.inferSchema                          false        false
2018-05-01T13:50:55.713+0545 INFORMATION kudu.inferSchemaPrefix                    -- ERROR --  -- ERROR --
```

Reply:
Right. I'd go for not having this logic here at all. Let the more "logical" class handle that.

Reply:
I prefer to provide senseful logging output.

Reply:
I see your point. It's simply doesn't seem to be a custom here -- eg `HiveClientConfig#isParquetPredicatePushdownEnabled` makes sense only if `HiveClientConfig#setParquetOptimizedReaderEnabled`, but there is no such `if` login there. 

(you can leave as-is though)